Winleafs.Api/Endpoints/ExternalControlEndpoint.cs

```diff
@@ -33,16 +33,18 @@ public async Task<ExternalControlInfo> GetExternalControlInfoAsync(DeviceType de
         {
             //For Aurora, we will use external control version 1 by passing no version
             //For Canvas, we will use external control version 2 by passing that version
-            switch (deviceType)
+            return (deviceType) switch
             {
-                case DeviceType.Aurora:
-                    return await SendRequestAsync<ExternalControlInfo>(BaseUrl, Method.PUT, body: "{\"write\": {\"command\": \"display\", \"animType\": \"extControl\"}}");
-                case DeviceType.Canvas:
-                case DeviceType.Shapes:
-                    return await SendRequestAsync<ExternalControlInfo>(BaseUrl, Method.PUT, body: "{\"write\": {\"command\": \"display\", \"animType\": \"extControl\", \"extControlVersion\": \"v2\"}}");
-                default:
-                    throw new NotImplementedException($"No external control info implemented for device type {deviceType.ToString()}");
-            }            
+                DeviceType.Aurora => await SendRequestAsync<ExternalControlInfo>(BaseUrl, Method.PUT,
+                    body: "{\"write\": {\"command\": \"display\", \"animType\": \"extControl\"}}"),
```

Comment:
Can this command be an object as well?

Reply:
Agreed and fixed, want te rework these into records but this will do for now.